common.js

```diff
@@ -195,8 +195,11 @@ function schemaToArray(schema,offset,options,data) {
     let container = [];
     let block = { title: '', rows: [] };
     if (schema) {
-        if (schema.description) block.title = schema.description;
-        if (!block.title && schema.title) block.title = schema.title;
+        if (schema.title) block.title = schema.title;
+        if (!block.title && schema.description)
+            block.title = schema.description;
+        else if (schema.description)
```

Comment:
Could we make this assignment to `block.description` unconditional on `!block.title` so it is always available in templates?

Reply:
Done 👍